<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/top_menu_50px.png</normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/top_menu_50px.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string> Add Folder</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/Add Folder_50px.png</normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/Add Folder_50px.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add">
            <property name="text">
             <string> Add File</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/plus_50px.png</normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/plus_50px.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_remove">
            <property name="text">
             <string> Remove</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/minus_50px.png</normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/minus_50px.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" rowspan="5">
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>List of images</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Predict all</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_predict">
            <property name="text">
             <string>Predict</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_clear">
            <property name="text">
             <string> Clear All</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/clear_symbol_50px.png</normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/clear_symbol_50px.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QLabel" name="result">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Result</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Rt and forecast</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>Kazakhstan</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Country name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="PlotWidget" name="graphicsView"/>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="PlotWidget" name="graphicsView_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset>
         <normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/info_50px.png</normaloff>../../.designer/yandex lyseum 2019-2020/imap-Image-to-PDF-Converter-Application-For-Windows-master/imap-Image-to-PDF-Converter-Application-For-Windows-master/icons/info_50px.png</iconset>
       </attribute>
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>191</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Technologies Used:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Saparov Almas </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Python 3.8, PyQt5, Tensorflow, Keras, cv2 and etc..</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Developer:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionRemove"/>
    <addaction name="actionAdd_folder"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove file</string>
   </property>
  </action>
  <action name="actionAdd_folder">
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
